src/Acuminator/Acuminator.Analyzers/StaticAnalysis/Descriptors.cs

```diff
@@ -111,11 +111,14 @@ public static class Descriptors
 		public static DiagnosticDescriptor PX1029_PXGraphUsageInDac { get; } =
 			Rule("PX1029", nameof(Resources.PX1029Title).GetLocalized(), Category.Default, DiagnosticSeverity.Error);
 
-		public static DiagnosticDescriptor PX1030_DefaultAttibuteToExisitingRecords { get; } =
-			Rule("PX1030", nameof(Resources.PX1030TitleDefaultAttributeOnDacExtension).GetLocalized(), Category.Default, DiagnosticSeverity.Warning);
+		public static DiagnosticDescriptor PX1030_DefaultAttibuteToExisitingRecordsError { get; } =
```

Comment:
mistype DefaultAttibuteToExis**i**tingRecordsError. Should be  PX1030_DefaultAttibuteToExistingRecordsError